.circleci/config.yml.in

```diff
@@ -288,6 +294,109 @@ jobs:
               aws s3 cp "$pkg" "s3://pytorch/whl/${UPLOAD_CHANNEL}/<< parameters.subfolder >>" --acl public-read
             done
 
+  smoke_test_linux_conda:
+    <<: *smoke_test_common
+    steps:
+      - attach_workspace:
+          at: ~/workspace
+      - run:
+          name: install binaries
+          command: |
+            set -x
+            source /usr/local/etc/profile.d/conda.sh && conda activate python${PYTHON_VERSION}
+            conda install -v -y -c pytorch-nightly pytorch
+            conda install -v -y $(ls ~/workspace/torchvision*.tar.bz2)
+      - run:
+          name: smoke test
+          command: |
+            source /usr/local/etc/profile.d/conda.sh && conda activate python${PYTHON_VERSION}
+            python -c "import torchvision"
```

Comment:
I think it would also be good to verify that the C++ extensions have been properly loaded.
This could be done with something like
```bash
python -c "import torchvision; assert torchvision._HAS_OPS" 
```
But maybe this can be sent in a follow-up PR?